src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,9 +33,21 @@
  */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private Analyze analyze = new Analyze();
```

Comment:
nit: add an empty line before and after this variable (and let's make it final too).